server/index.js

```diff
@@ -1,12 +1,52 @@
 require('dotenv/config');
+const pg = require('pg');
 const express = require('express');
+const ClientError = require('./client-error');
 const errorMiddleware = require('./error-middleware');
 const staticMiddleware = require('./static-middleware');
 
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
 const app = express();
 
+const jsonMiddleware = express.json();
+app.use(jsonMiddleware);
+
 app.use(staticMiddleware);
 
+app.post('/api/lists/new-list', (req, res, next) => {
```

Comment:
You can leave out the `new-list` part of the route path, because "new list" is implied by `POST`.